wild/tests/integration_tests.rs

```diff
@@ -2639,18 +2692,8 @@ fn read_test_config() -> Result<TestConfig> {
         );
     };
 
-    // The environment variable can override the config file setting.
-    if let Ok(cross_arch) = std::env::var("WILD_TEST_CROSS") {
-        config.qemu_arch = cross_arch
-            .split(',')
-            .map(|s| s.trim())
-            .filter(|s| !s.is_empty())
-            .map(|s| {
-                Architecture::from_str(s)
-                    .with_context(|| format!("Unknown WILD_TEST_CROSS value `{s}`"))
-            })
-            .collect::<Result<Vec<Architecture>>>()?
-    }
+    // The environment variable `WILD_TEST_CROSS` can override the config file setting.
+    config.qemu_arch = get_wild_test_cross();
```

Comment:
It looks like the value of qemu_arch from the config file will now always be ignored. Perhaps return an `Option<Vec<Architecture>>` from `get_wild_test_cross`. Then the other caller can use `unwrap_or_default` on the result, while this call-site can conditionally assign.